modules/consensus/consensusset.go

```diff
@@ -61,6 +61,10 @@ type ConsensusSet struct {
 	// marshaler encodes and decodes between objects and byte slices.
 	marshaler encoding.GenericMarshaler
```

Comment:
You want it like this?

```go
 	// marshaler encodes and decodes between objects and byte slices.
 	marshaler encoding.GenericMarshaler
	// blockRuleHelper calculates values on blocks for block validity rule
	// enforcement.
	blockRuleHelper blockRuleHelper
```

I don't have a strong objection to it because it's just whitespace but it seems like a strange request because it would make the spacing inconsistent with the rest of the struct.